Gemfile.lock

```diff
@@ -27,19 +27,6 @@ GIT
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
-    rest-client (2.1.0-x64-mingw-ucrt)
-      ffi (~> 1.15)
-      http-accept (>= 1.7.0, < 2.0)
-      http-cookie (>= 1.0.2, < 2.0)
-      mime-types (>= 1.16, < 4.0)
-      netrc (~> 0.8)
```

Comment:
This looks like we're dropping REST from Windows platform. I know that we have a bunch of changes non-responsive-upstream hasn't accepted, is that why? How is Chef on windows getting a REST client now?

Reply:
That was a total gaffe. Updating the gemfile.lock on Windows is a total crap-shoot sometimes. Now fixed